frontend/catalyst/jit.py

```diff
@@ -212,67 +214,6 @@ def circuit(x: int):
         be unrolled during tracing, "copy-pasting" the body 5 times into the program rather than
         appearing as is.
 
-
```

Comment:
I removed these as the qjit-docstring was quite long, and they seemed out of place (no other specific autograph features are documented like this). Instead, better to link users to the autograph guide rather than duplicating all this documentation.